components/base/agilex/limo_base.go

```diff
@@ -330,12 +330,19 @@ func (lb *limoBase) setMotionCommand(linearVel float64,
 	return nil
 }
 
+// positive angleDeg spins base left. degsPerSec is a positive angular velocity.
```

Comment:
could you return immediately with an error if a zero degsPersec is passed in? Just noticing the bug

Reply:
in line 336